autonomi/src/client/data_types/scratchpad.rs

```diff
@@ -139,12 +148,14 @@ impl Client {
     }
 
     /// Create a new scratchpad to the network
+    /// Returns the cost of the scratchpad and the address of the scratchpad
     pub async fn scratchpad_create(
         &self,
         scratchpad: Scratchpad,
         payment_option: PaymentOption,
-    ) -> Result<AttoTokens, PutError> {
+    ) -> Result<(AttoTokens, ScratchpadAddress), PutError> {
```

Comment:
Same comment as on the `chunk_upload_with_payment`. Why return the address?

Reply:
Yeah trying for all `put`s to return price + address. 
Even though we know that `Scratchpad::address` also does the trick, I think it's a user friendly API to return the address were it was put